scipy/stats/tests/test_kdeoth.py

```diff
@@ -396,6 +396,7 @@ def test_pdf_logpdf_weighted():
     assert_almost_equal(pdf, pdf2, decimal=12)
 
 
+@pytest.mark.slow
```

Comment:
@ilayn this test was taking 1.36s on my machine. Is it OK to mark it slow?

Reply:
Yes either that or we can change the size of the arrays to make it more acceptable. Either way is fine since we tested this enough times.

Reply:
Since it was for "very high dimensionality kde" i figured that it would inherently take a while. Let's mark it slow.